### PR TITLE
Add jq package needed for artifacts processing

### PR DESCRIPTION
**Pull request by cevich**
**- What I did**

Add 'jq' package.

**- How I did it**

Added it here so it gets built into the AMI's as per @stevekuznetsov 

>  runcom, mrunalp, cevich looks like you need `jq` on your AMIs to push the artifacts

**- How to verify it**

With ``/test ami`` (regular tests won't do it), then check Origin CI logs for error posting to GCE.

### CODE REVIEW

**Comment by cevich** [comment]
***Note:*** I'm blindly assuming the package is available.  This could also be done in the Origin CI 'sjb' files, but there are many more of those to change than doing it here.

**Comment by mrunalp** [comment]
@cevich We can merge this if you can confirm that the tasks got past the jq install phase.

**Comment by mrunalp** [comment]
Definitely not present in configure RHEL repos:
```
failed: [localhost] (item=jq) => {
    "ansible_job_id": "651927027144.4474", 
    "changed": false, 
    "finished": 1, 
    "item": "jq", 
    "rc": 126, 
    "results": [
        "No package matching 'jq' found available, installed or updated"
    ]
}
```

**Comment by runcom** [comment]
> Definitely not present in configured RHEL repos:

let's check what Origin does in its AMIs on RHEL, they must be enabling some repo

**Comment by cevich** [comment]
update: In Fedora, ``jq`` is available.  In RHEL/CentOS we need to enable EPEL.  But I want to see the ``ami_fedora`` result first, in case there are other changes needed.

**Comment by cevich** [comment]
...though, we don't actually need ``ami_fedora`` to pass.  So can I ssh into the VM and break crio or the tests to hurry things along?  It should still attempt the GCE things, ya?

**Comment by mrunalp** [comment]
@cevich You can look at the jenkins console. Also, this is on AWS on GCE. Feel free to ssh and muck around :)

**Comment by cevich** [comment]
Note: The ``ami_rhel`` test failed for an odd-ball reason, seemingly unrelated to any changes here.  Not sure what to make of it.

```
...snip...

TASK [ensure directories exist as needed] **************************************
task path: /go/src/github.com/kubernetes-incubator/cri-o/contrib/test/integration/system.yml:102
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'Task' object has no attribute 'async_val'
fatal: [localhost]: FAILED! => {}

MSG:
---
Unexpected failure during module execution.
---
	to retry, use: --limit @/go/src/github.com/kubernetes-incubator/cri-o/contrib/test/integration/main.retry
```

File: contrib/test/integration/main.yml [hunk 1]
```diff
@@ -118,3 +122,16 @@
...
     - name: run k8s e2e tests
       include: e2e.yml
+
+# N/B: This is done after all testing, so it doesn't get in the way.
+- hosts: all
+  remote_user: root
+  vars_files:
+    - "{{ playbook_dir }}/vars.yml"
+  tags:
+    - always
```
Comment (runcom) [inline contrib/test/integration/main.yml]: this requires a change in another repository to make it working, have you already created another patch on said repo?

Comment (runcom) [reply contrib/test/integration/main.yml]: I don't remember if it's openshift/release or aos-cd-jobs, please check that and open the relevant PR over there so we can land this asap

Comment (cevich) [reply contrib/test/integration/main.yml]: The `always` tag is magic.  It "always" runs no matter what tags are specified on command-line.  Last I checked this part was working as written, not sure about latest runs, will check soon.

**Comment by cevich** [comment]
@runcom last failures are flakes or breakage related to installing docker (somehow).  Changed up order of operations, we'll see if that helps...

**Comment by cevich** [comment]
/me is watching job consoles.  All good so far, no more ansible weirdness.

**Comment by cevich** [comment]
:confused: this way is getting too complicated.  If anything happens during the playbook run, like a testing failure/flake, docker doesn't start and so no results are posted.  This means we only ever post positive results...which is nice, but unhelpful.

Going to switch gears here and try installing epel, jq, docker then starting the daemon as post-tasks in aos-cd-jobs.  It's less ideal, but somewhat simpler and doesn't depend on the playbook completing.

Leaving this PR open for the short-term, in case I need to come back to it...

**Comment by cevich** [comment]
X-ref: https://github.com/openshift/aos-cd-jobs/pull/1569

**Pull request closed without merging**